cmd/agent/subcommands/run/settings.go

```diff
@@ -35,5 +35,15 @@ func initRuntimeSettings(serverDebug dogstatsddebug.Component) error {
 	if err := commonsettings.RegisterRuntimeSetting(commonsettings.NewProfilingGoroutines()); err != nil {
 		return err
 	}
+	if err := commonsettings.RegisterRuntimeSetting(
+		settings.NewHighAvailabilityRuntimeSetting("ha.enabled", "Enable/disable High Availability support."),
+	); err != nil {
+		return err
+	}
+	if err := commonsettings.RegisterRuntimeSetting(
+		settings.NewHighAvailabilityRuntimeSetting("ha.failover", "Enable/disable redirection of telemetry data to failover region."),
+	); err != nil {
+		return err
+	}
```

Comment:
Could you specify what the possible values (true / false) are in the description ?

Reply:
Will do. 👍🏻